bgpd/bgp_zebra.c

```diff
@@ -1421,13 +1423,20 @@ void bgp_zebra_announce(struct bgp_node *rn, struct prefix *p,
 			}
 
 			label_buf[0] = '\0';
+			eth_buf[0] = '\0';
 			if (has_valid_label
 			    && !CHECK_FLAG(api.flags, ZEBRA_FLAG_EVPN_ROUTE))
 				sprintf(label_buf, "label %u",
 					api_nh->labels[0]);
-			zlog_debug("  nhop [%d]: %s if %u VRF %u %s",
+			if (CHECK_FLAG(api.flags, ZEBRA_FLAG_EVPN_ROUTE)
+			    && !is_zero_mac(&api_nh->rmac))
+				snprintf(eth_buf, ETHER_ADDR_STRLEN + 7,
+					 " RMAC %s",
+					 prefix_mac2str(&api_nh->rmac,
+							buf1, sizeof(buf1)));
```

Comment:
hmm - still not quite consistent:

use `snprintf` for both label_buf and eth_buf

just do `sizeof` for all of label_buf, buf1, and eth_buf